src/morale_types.h

```diff
@@ -5,6 +5,7 @@ enum morale_type : int {
     MORALE_NULL = 0,
     MORALE_FOOD_GOOD,
     MORALE_FOOD_HOT,
+    MORALE_FOOD_CARNIVORE,
```

Comment:
Right, indeed. What is a better way to edit the enums? (I'll redo the morale bonus a different way, so just hypothetically).